ferrocene/tools/self-test/src/targets.rs

```diff
@@ -52,6 +52,15 @@ impl Deref for Target {
     }
 }
 
+impl TargetSpec {
+    fn expected_libs(&self) -> &[&str] {
```

Comment:
me would keep this inlined
- there is only one user
- is trivial

Reply:
Reverted it